libs/locales/package.json

```diff
@@ -51,5 +51,5 @@
     "lint": "yarn run -T eslint --cache --cache-location node_modules/.cache/eslint/.eslint-cache ."
   },
   "type": "module",
-  "version": "1.0.0"
+  "version": "2.31.0"
```

Comment:
Do we usually update the versions of our libs? Isn't that done automatically during the releases?

Reply:
It looks like this change is coming from the first commit, could it be that the wrong branch / commit was used as the base?

Reply:
@jgyselov @celdrake the base branch was incorrect